src/general/StageBase.cs

```diff
@@ -196,10 +199,25 @@ public virtual void ResolveNodeReferences()
         NodeReferencesResolved = true;
     }
 
+    public override void _Ready()
+    {
+        base._Ready();
+
+        // Need to check for null light cycle in case this is loaded from an old save
```

Comment:
Please use the save load notification callbacks instead of doing this in _Ready.

Reply:
By this do you mean Setup Stage or is there something I'm missing?

Reply:
Yes you are missing use of `ISaveLoadable` interface.